cpp/daal/src/algorithms/covariance/covariance_hyperparameter.cpp

```diff
@@ -21,6 +21,7 @@
 //--
 */
 
+#include <stdint.h>
```

Comment:
It contradicts usage of `DAAL_INT64`

Reply:
It is there for uint32_t - a type of the keys in hash tables. I do not see the contradiction.